server/routes/POST-upload.js

```diff
@@ -24,5 +25,6 @@ module.exports = (app, s3Client) => {
         console.log("\nUploaded Object:\n", data);
       }
     });
+	res.send({});
```

Comment:
Can you fix the indent here. Also, can we include a status that indicates the request is successful?